src/opera/util/input_validation.py

```diff
@@ -33,6 +33,8 @@ def check_input(input_object, logger, name, valid_extensions=None):
     valid_extensions : iterable, optional
         Expected file extensions of the file being validated. If not provided,
         no extension checking will take place.
+    check_zero_size : boolean, optional
+        If true, raise an exception for zero-size input objects
```

Comment:
Might be a little clearer to say something like:  "If true, test for zero-size input."